src/browser/modules/D3Visualization/lib/visualization/components/Node.ts

```diff
@@ -73,15 +80,9 @@ export default class NodeVisualisationModel {
     return this.propertyMap
   }
 
-  relationshipCount(graph: any): number {
-    let count = 0
-    for (const relationship of Array.from(graph.relationships())) {
-      if (
-        (relationship as any).source === this ||
-        (relationship as any).target === this
-      )
-        count++
-    }
-    return count
+  relationshipCount(graph: Graph): number {
+    return graph
+      .relationships()
+      .filter(rel => rel.source === this || rel.target === this).length
```

Comment:
👍

Reply:
If d3 changes how it does mutation, might want to compare ids instead here in the future